General queries/WD AV Signature and Platofrm Version.txt

```diff
@@ -0,0 +1,15 @@
+//Author: António Vasconcelos
+//Twitter: https://twitter.com/anthonws
+// This query provides you the latest signature and platform (MoCamp) for Windows Defender AV
+FileCreationEvents
+| where InitiatingProcessCommandLine contains "MpSigStub.exe"
+| extend NewVersion=tostring(split(InitiatingProcessCommandLine, " ")[4]) 
+| summarize arg_max(NewVersion, *) by ComputerName
+| project ComputerName , NewVersion
+| join (FileCreationEvents
+        | where FileName == "MsMpEng.exe"
+        | where FolderPath contains @"C:\ProgramData\Microsoft\Windows Defender\Platform\"
```

Comment:
Performance: I think `startswith` can be applied here. For string operators `contains` should be the last option, perf-wise

Reply:
Agreed.